jib-core/src/main/java/com/google/cloud/tools/jib/image/ImageReference.java

```diff
@@ -167,38 +186,72 @@ public static boolean isValidRepository(String repository) {
   }
 
   /**
+   * Returns {@code true} if {@code tag} is a valid tag string. For example, a valid tag could be
+   * {@code v120.5-release}.
+   *
    * @param tag the tag to check
    * @return {@code true} if is a valid tag; {@code false} otherwise
    */
   public static boolean isValidTag(String tag) {
     return tag.matches(TAG_REGEX);
   }
 
+  /**
+   * Returns {@code true} if {@code tag} is the default tag (@code latest}; {@code false} if not.
```

Comment:
"is the default tag latest or empty"? BTW, I'm not sure if it makes sense that this method returns true for an empty input as a utility method named `isDefaultTag`.

Reply:
Hmm, I'm thinking that `default` means that it would be the default if the tag was empty.